src/aws_sdk_dynamodbstore.rs

```diff
@@ -474,98 +693,68 @@ impl super::Backend for Backend {
             .unwrap_or(HashMap::new()))
     }
 
+    #[tracing::instrument(skip_all, fields(key, consumed_wcu, otel.status_code, error.msg, otel.span_kind = "client"))]
+
```

Comment:
style nit: why the extra newline between the `#[...]` and `async fn ...` here and below?

Reply:
Weird pasting, I guess. Not sure why I never fixed it.